SS23-SWEN2-TourPlanner/Views/ModalPage.xaml.cs

```diff
@@ -32,6 +32,8 @@ private void AddTour_OnClicked(object sender, EventArgs e)
         var tour = (Tour)BindingContext;
         Console.WriteLine("Add Tour");
         
+        tour.Information = System.IO.Path.Join(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "TourPlanner", "Maps", $"{tour.Name}.png");
+        
```

Comment:
Das würde ich nicht in der View machen, das ist schon zu viel Logik für eine View. Theoretisch müssten doch beim generieren der Map einfach die Information gesetzt werden. 
BTW finde ich das Wording für die Map eher schlecht. Wir sollten das unbedingt auf "Map", "MapImage" oder so umändern.